### PR TITLE
Improve number of repetitions and result description

### PR DESCRIPTION
**Pull request by Algodev-github**
Improve number of repetitions and result description

### CODE REVIEW

File: automated/linux/ssuite/run-bench.sh [hunk 1]
```diff
@@ -67,6 +68,9 @@ usage() {
 	If this parameter is set to yes then only workloads made of
 	only reads are generated in every benchmark.
 	Default value: no"
+
+	<NUM_REPETITIONS>:
+	Number of times each benchmark is repeated. Default: 5.
```
Comment (mwasilew) [inline automated/linux/ssuite/run-bench.sh]: closing quotation mark needs to be moved.

File: automated/linux/ssuite/run-bench.sh [hunk 2]
```diff
@@ -174,7 +178,8 @@ run_test() {
 	rm -f ${HOME_DIR}/.S-config.sh
 
 	cd "$S_PATH"/run_multiple_benchmarks/ || exit 1
-	./run_main_benchmarks.sh "$1" "" "" "" "${ONLY_READS}" "" 5 "${OUTPUT}" 2>&1 |\
+	./run_main_benchmarks.sh "$1" "" "" "" "${ONLY_READS}" "" \
+				 ${NUM_REPETITIONS} "${OUTPUT}" 2>&1 |\
```
Comment (mwasilew) [inline automated/linux/ssuite/run-bench.sh]: how is the NUM_REPETITIONS parameter updated from user params in this script?

**Comment by mwasilew** [comment]
This looks good. If you could just fix shellcheck warning:
```
* SHELLCHECK: [FAILED]: ./automated/linux/ssuite/run-bench.sh
* SHELLCHECK: [OUTPUT]:
 In ./automated/linux/ssuite/run-bench.sh line 186:
 				 ${NUM_REPETITIONS} "${OUTPUT}" 2>&1 |\
                                  ^----------------^ SC2086: Double quote to prevent globbing and word splitting.
```

**Comment by Algodev-github** [comment]
Great! I've just finished testing a fixed version. I'm about to fix this last issue too, and then submit a new PR.

Thanks,
Paolo

**Comment by Algodev-github** [comment]
Fixes merged in this PR!
Patches tested succesfully:
https://lkft.validation.linaro.org/scheduler/job/1273727

**Pull request merged**